datastore/cloud-client/query_filter_or.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START datastore_query_filter_or]
+from google.cloud import datastore
+from google.cloud.datastore import query
+
+"""
+Builds a union (OR) query filter.
```

Comment:
Move outside region tags, single whitespace after license header and two whitespace before START tag